src/main/java/edu/njit/jerse/automation/RepositoryAutomationEngine.java

```diff
@@ -39,6 +44,9 @@ private record Repository(
         }
     }
 
+    // TODO: Do we want to consider adding the LLM model in the CSV file?
+    // TODO: Users would be able to specify the model to use for each repository.
+    // TODO: This would be beneficial if we wanted to run multiple different models on the same repository.
```

Comment:
This is a good idea! (But, we shouldn't do it until we need it.)

Reply:
Converting this to an Issue - #29